### ISSUE TITLE
sink output

### ISSUE THREAD

**Issue by majedbawarshi** [IniasP/rnnoise-cli#6]
hello again, firstly I would like to thank you for the effort you have put into developing this package.

I have noticed when activating the plugin, it automatically generates 2 sink output devices; I would appreciate it if you would enlighten me about the purpose of generating them; would I face any issue if I canceled the creation of them? and how may I cancel the generation of the sink output?

**Comment by IniasP** [comment]
One of them takes the input stream to the noise suppression plugin and feeds filtered output into the other one.
This application basically automates the instructions at https://github.com/werman/noise-suppression-for-voice#pulseaudio, you can read more there and even check out his further reading links. [This one](https://bugs.freedesktop.org/show_bug.cgi?id=101043) directly explains what's happening.

Once the output sink exists, some applications will be able to select something like "Monitor of RNNoise Denoised Sink" (which is the one the plugin sink outputs to) directly as input device, but most won't (as noted in werman's instructions, most notably Chrome and Chromium-based browsers, which includes Electron apps like Discord). Because of that, I do the remapping step mentioned in the instructions as well, to create a proper input device through `module-remap-source`.

**TLDR:** the sinks are necessary.
